### PR TITLE
create avatar, badge, image component

### PR DESCRIPTION
**Pull request by git-0r**
# Created avatar, badge, image component.

1. Avatar includes following types of avatars:
 - Rounded avatar
 - Avatar with status icon
 - Avatar with text

avatar

2. Badges are of two types:
 - badge on avatar
 - badge on icon
 
badge1
badge2

3. Images are of two types:
 - Responsive images
 - Round Images

image1
image2

### CODE REVIEW

**Review by Naman-Saxena1 — APPROVED** [review]
Looks good to merge, can improve commit messages by using proper prefixes.

File: css/comp_lib_styles/utils/typography.css [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+h2 {
+    font-size: 2rem;
+    letter-spacing: -0.5px;
+    line-height: 1.5em;
+}
+
+h3 {
+    font-size: 1.5rem;
+    letter-spacing: 0px;
```
Comment (varun1233) [inline css/comp_lib_styles/utils/typography.css]: Suggestion: you can directly use 0

File: pages/comp_image.html [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+                    <li class="menu__item"><a class="menu__link" href="./comp_image.html">Image</a></li>
+                </ul>
+            </details>
+        </nav>
+        <section class="docs__section box-shadow">
+            <article class="docs__content">
+                <h1 id="image">Image</h1>
+                <h2 id="responsive-image">Responsive image</h2>
+                <div class="component__container">
+                    <div style="width: 300px; border-radius: 0.25rem; overflow: hidden;">
```
Comment (varun1233) [inline pages/comp_image.html]: Don't use inline styles

File: style.css [hunk 3]
```diff
@@ -0,0 +1,139 @@
...
+.box-shadow {
+  box-shadow: 0px 24px 38px rgba(0, 0, 0, 0.06),
+    0px 9px 46px rgba(0, 0, 0, 0.04), 0px 11px 15px rgb(0, 0, 0, 0.08);
+}
+
+.docs__section {
+  width: 60%;
+  margin: 1.5rem 0 1rem 0;
+  border-radius: var(--border-radius-sm);
+  background-color: #fbfaf5;
```
Comment (varun1233) [inline style.css:27]: use variables

**Review by Sahil25061999 — APPROVED** [review]
Most of the code looks good, nice use of comments . Just a suggestion to use px if value is less than 10 for the badge.css file, rest looks good to me.

**Pull request merged**